bundles/org.openhab.binding.dbquery/README.md

```diff
@@ -0,0 +1,213 @@
+| resultSwitch    | Switch    | Result of last executed query as Switch, query must have `scalarResult=true` |
+| parameters      | String    | Contains parameters of last executed query as JSON|
+| correct         | Switch    | `ON` if the result of last executed query was correct |
+
+All the channels, except `execute`, are updated when the query execution finishes, and while there is a query in execution they have the values from
+last previous executed query.
+
+The `resultString` channel is the only valid one if `scalarResult=false`, and in that case it contains the query result serialized to JSON in that format:
+
+    {
+        correct : true,
+        data : [
+            { 
+                column1 : value,
+                column2 : value
+            },
+            { ... }, //row2        
+            { ... }  //row3
+        ]
+    }
+    
+### Channel Triggers
+
+#### calculateParameters
+
+Triggers when there's a need to calculate parameters before query execution.
+When a query has `hasParameters=true` it fires the `calculateParameters` channel trigger and pause the execution until `setQueryParameters` action is call in
+ that query.
+ 
+In the case a query has parameters it's expected that there is some rule that catch the `calculateParameters` trigger, calculate the parameters
+ with the corresponding logic and then call the `setQueryParameters` action, after that the query will be really executed.
+ 
+## Actions
+
+### For DatabaseBridge
+
+#### executeQuery 
+
+It allows executing a query synchronously from a script/rule without defining it in a Thing.
+
+To execute the action you need to pass the following parameters:
+
+- String query: The query to execute
+- Map<String,Object>: Query parameters (empty map if not needed)
+- int timeout: Query timeout in seconds
+
+And it returns an `ActionQueryResult` that has the following properties:
+
+- correct (boolean) : True if the query was executed correctly, false otherwise
+- data (List<Map<String,Object>>): A list where each element is a row that is stored in a map with (columnName,value) entries  
+- isScalarResult: It returns if the result is scalar one (only one row with one column)
+- resultAsScalar: It returns the result as a scalar if possible, if not returns null
+
+
+Example (using Jython script):
+
+     from core.log import logging, LOG_PREFIX 
+     log = logging.getLogger("{}.action_example".format(LOG_PREFIX))
+     map = {"time" : "-2h"}
+     influxdb = actions.get("dbquery","dbquery:influxdb2:sampleQuery") //Get bridge thing
+     result = influxdb.executeQuery("from(bucket: \"default\") |> range(start:-2h)  |> filter(fn: (r) => r[\"_measurement\"] == \"go_memstats_frees_total\")  |> filter(fn: (r) => r[\"_field\"] == \"counter\")  |> mean()",{},5)
+     log.info("execute query result is "+str(result.data))
+    
+
+Use this action with care, because as the query is executed synchronously is not good to execute long-running queries and block script execution.
```

Comment:
```suggestion
Use this action with care, because as the query is executed synchronously, it is not good to execute long-running queries that can block script execution.
```